### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/CorsEndpointProperties.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 	/**
 	 * Comma-separated list of origin patterns to allow. Unlike allowed origins which only
-	 * supports '*', origin patterns are more flexible (for example
+	 * support '*', origin patterns are more flexible (for example
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/CorsEndpointProperties.java:49]: I think "supports" is correct here.

Comment (izeye) [reply spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/CorsEndpointProperties.java:49]: @wilkinsona "origins" is plural, so I thought it needs to be "support". Am I missing something?

Comment (wilkinsona) [reply spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/CorsEndpointProperties.java:49]: While "allowed origins" is plural, it's referring to the allowed origins property and there's one of those.

Comment (izeye) [reply spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/CorsEndpointProperties.java]: @wilkinsona Thanks for the explanation! I reverted it.

**Pull request closed without merging**